ckanext/dcat/harvesters/_json.py

```diff
@@ -382,3 +381,9 @@ def copy_across_resource_ids(existing_dataset, harvested_dataset):
                     matching_existing_resource)
         if not existing_resources_still_to_match:
             break
+
+    config = config if config is not None else {}
+    if config.get('keep_manual_added_resources', False):
```

Comment:
I think it would be better to rename the config `keep_manual_added_resources` to `keep_existing_resources`.